web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp

```diff
@@ -15,318 +15,322 @@
-	 <c:otherwise> <fmt:message key="na" bundle="${resword}"/></c:otherwise></c:choose><br></td>
-</tr>
-<tr>
- <td class="table_cell_noborder" ><b><fmt:message key="event" bundle="${resword}"/>:</b></td>
- <td class="table_cell_noborder" ><c:out value="${studyEvent.studyEventDefinition.name}"/> (<fmt:formatDate value="${studyEvent.dateStarted}" pattern="${dteFormat}"/>)</td>
- <td class="table_cell_top" ><b><fmt:message key="date_of_birth" bundle="${resword}"/>:</b><br></td>
- <td class="table_cell_noborder" ><fmt:formatDate value="${subject.dateOfBirth}" pattern="${dteFormat}"/><br></td>
-</tr>
-<tr>
-  <td class="table_cell_noborder" ><b><fmt:message key="interviewer" bundle="${resword}"/>:</b></td>
-  <td class="table_cell_noborder" ><c:out value="${EventCRFBean.interviewerName}"/> (<fmt:formatDate value="${EventCRFBean.dateInterviewed}" pattern="${dteFormat}"/>)</td>
-  <td class="table_cell_top" ><b><fmt:message key="Gender" bundle="${resword}"/>:</b></td>
-  <td class="table_cell_noborder" ><c:choose>
-    <c:when test="${subject.gender==109}"><fmt:message key="M" bundle="${resword}"/></c:when>
-    <c:when test="${subject.gender==102}"><fmt:message key="F" bundle="${resword}"/></c:when>
-    <c:otherwise><c:out value="${subject.gender}"/></c:otherwise>
-</c:choose></td>
-</tr>
-</table>
-
-
-
-<span ID="spanAlert-interviewDate" class="alert"></span>
-</td>
-				</tr>
-			</table>
-
-	  	</td>
-	</tr>
-				</table>
-
-		</div>
-
-	</div></div></div></div></div></div></div>
+    <table border="0" cellpadding="0" cellspacing="0">
+        <tr id="CRF_infobox_closed" style="display: none;">
+            <td style="padding-top: 3px; padding-left: 6px; width: 90px;" nowrap>
+                <a href="javascript:leftnavExpand('CRF_infobox_closed'); leftnavExpand('CRF_infobox_open');"><img src="images/sidebar_expand.gif" border="0" align="right" hspace="10"></a>
+
+                <b>CRF Info</b>
+
+            </td>
+        </tr>
+        <tr id="CRF_infobox_open" style="display: all">
+
+            <td>
+                <table border="0" cellpadding="0" cellspacing="0">
+                    <tr>
+                        <td valign="bottom">
+                            <table border="0" cellpadding="0" cellspacing="0" width="100">
+                                <tr>
+                                    <td nowrap>
+                                        <div class="tab_BG_h">
+                                            <div class="tab_R_h" style="padding-right: 0px;">
+                                                <div class="tab_L_h" style="padding: 3px 11px 0px 6px; text-align: left;">
+                                                    <a href="javascript:leftnavExpand('CRF_infobox_closed'); leftnavExpand('CRF_infobox_open');"><img src="images/sidebar_collapse.gif" border="0" align="right"></a>
+
+                                                    <b>CRF Info</b>

```

Comment:
If you are removing hardcoded strings in this file, this one should also be added to a properties file.